server/event_logging/tests/test_event.py

```diff
@@ -0,0 +1,10 @@
+from ..event import Event
```

Comment:
reference import always from the server module level. Somewhow Python is very stupid in handling imports. This way we keep it clean and consistent.

`from event_logging.event import Event`

Reply:
Apparently, we have some issue regarding my adjustments of the test step in our CI.

Reply:
Apparently, we have some issue regarding my adjustments of the test step in our CI.